src/ralph/models/xapi/video/statements.py

```diff
@@ -161,28 +167,66 @@ class VideoTerminated(BaseVideoStatement):
         verb__id="http://adlnet.gov/expapi/verbs/terminated",
     )
 
-    verb: VideoTerminatedVerbField
-    result: VideoTerminatedResultField
-    context: VideoTerminatedContextField
+    verb: VideoTerminatedVerbField = VideoTerminatedVerbField()
+    result: Optional[VideoTerminatedResultField]
+    context: Optional[VideoTerminatedContextField]
 
 
-class VideoInteracted(BaseVideoStatement):
-    """Represents a video terminated xAPI statement.
+class VideoEnableClosedCaptioning(BaseVideoStatement):
```

Comment:
Missing `__selector__` in `VideoEnableClosedCaptioning`, `VideoVolumeChangeInteraction` and `VideoScreenChangeInteraction` ?

Reply:
The issue for the three model is that they have the same signatures for the selector...

Reply:
#187 should solve this issue (is ready for review), could we pass it before this pull request?